openhub/outside_committers.py

```diff
@@ -0,0 +1,64 @@
+import json
+import requests
+import logging
+import xmltodict
+
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+from openhub.models import OutsideCommitter, ContributionsToPortfolioProject
+
+
+def get_outside_committers_data():
+    n = 100
+    data1 = []
+    for i in range(1, n):
+        import_url = 'https://www.openhub.net/orgs/' + get_owner() \
```

Comment:
create a function to build this URL, as the URL is very similar for each of them.

Also the page iteration process is very similar, so should be possible to create a generic function to get the data into a JSON like dict .

Also, looks like you could use https://github.com/jpopelka/openhubinfo to fetch the data as XML.  You probably need to add 'org' support to that script.  We can use your fork of that script to create JSON files, and then use your existing code code to import those json files.